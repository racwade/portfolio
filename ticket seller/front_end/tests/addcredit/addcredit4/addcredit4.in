login
test_admin
addcredit
100
test_buy
logout