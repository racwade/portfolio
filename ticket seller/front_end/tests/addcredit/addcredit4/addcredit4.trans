06 test_buy      BS 000001099
00 test_buy		 BS 000001099