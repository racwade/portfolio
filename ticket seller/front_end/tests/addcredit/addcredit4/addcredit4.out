welcome to the ticket seller system
enter username:
logged in as test_admin
enter event name:
enter credit amount:
enter user to add credit to:
thank you, $100 added to test_buy
bye